assets/js/googlesitekit/datastore/user/user-info.test.js

```diff
@@ -108,21 +108,23 @@ describe( 'core/user userInfo', () => {
 				);
 			} );
 		} );
-		describe( 'receiveUserInputState', () => {
-			it( 'requires userInputState param', () => {
+		describe( 'receiveIsUserInputCompleted', () => {
+			it( 'requires isUserInputCompleted param', () => {
 				expect( () => {
-					registry.dispatch( CORE_USER ).receiveUserInputState();
-				} ).toThrow( 'userInputState is required.' );
+					registry
+						.dispatch( CORE_USER )
+						.receiveIsUserInputCompleted();
+				} ).toThrow( 'isUserInputCompleted is required.' );
 			} );
 
 			it( 'receives and sets userInputData', async () => {
-				const { userInputState } = userData;
+				const { isUserInputCompleted } = userData;
 				await registry
 					.dispatch( CORE_USER )
-					.receiveUserInputState( userInputState );
+					.receiveIsUserInputCompleted( isUserInputCompleted );
 				expect(
-					registry.select( CORE_USER ).getUserInputState()
```

Comment:
Tests for actions should make assertions using the store's state rather than using the selector which has its own tests below. E.g. https://github.com/google/site-kit-wp/blob/6830cd065166e62ac6dab6c3f096c6e0f5f37792/assets/js/googlesitekit/modules/datastore/sharing-settings.test.js#L277-L286